pkg/controller/kieapp/kieapp_controller_test.go

```diff
@@ -48,7 +47,7 @@ func TestUnknownEnvironmentObjects(t *testing.T) {
 	env, err := defaults.GetEnvironment(cr, test.MockService())
 	assert.Equal(t, fmt.Sprintf("envs/%s.yaml does not exist, '%s' KieApp not deployed", cr.Spec.Environment, cr.Name), err.Error())
 
-	env = consolidateObjects(env, cr)
+	env = defaults.ConsolidateObjects(env, cr)
```

Comment:
Move to defaults_test.go. It is only testing the ConsolidateObjects function.
Applies to
* TestUnknownEnvironmentObjects
* TestTrialConsoleEnv
* TestTrialServerEnv
* TestTrialServersEnv